src/index.html

```diff
@@ -17,16 +17,22 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form">
```

Comment:
Consider using a more semantic tag like <section> or <div> with a role attribute for the form container to improve accessibility and SEO.